### PR TITLE
updates for MGMT-297

### PR DESCRIPTION
**Pull request by aireilly**
Addresses https://issues.redhat.com/browse/MGMT-297, docs delivered in https://issues.redhat.com/browse/TELCODOCS-62

### CODE REVIEW

**Comment by swopebe** [comment]
@cadawson it seems a PR was created from outside of a GitHub issue but not assigned... Is this something you are aware of or can cover/validate? Assigning you for now, please let me know if you cannot.

File: clusters/bare_assets.adoc [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
-Use the following procedure to create a bare metal asset that can host a cluster that is managed by {product-title}.
+The hub cluster inventory controller defines a custom resource definition (CRD) called `BareMetalAsset`, which is used to hold a bare metal asset inventory record. When provisioning a managed cluster, the inventory controller reconciles `BareMetalAsset` inventory records with corresponding `BareMetalHost` resources in the managed cluster.
+
+`BareMetalAsset` CRs are intended to be used to provision cluster hardware based on records entered in a configuration management database (CMDB) or similar system. An external tool or automation polls the CMDB and uses the {product-title-short} API to create corresponding `BareMetalAsset` and corresponding `Secret` resources in a hub cluster for subsequent deployment in a managed cluster.
+
+Use the following procedures to create and manage bare metal assets for clusters managed by {product-title}.
 
 * <<bma-prerequisites,Prerequisites>>
 * <<creating-a-bare-metal-asset-with-the-console,Creating a bare metal asset with the console>>
+* <<creating-a-bare-metal-asset-with-oc,Creating a bare metal asset using the oc CLI>>
```
Comment (swopebe) [inline clusters/bare_assets.adoc]: Let's not do `oc` in the title here.

**Comment by aireilly** [comment]
> @cadawson it seems a PR was created from outside of a GitHub issue but not assigned... Is this something you are aware of or can cover/validate? Assigning you for now, please let me know if you cannot.

Yes, I discussed this with Chris. Let me know if there is anything else I can do from my side to help progress this.

File: clusters/bare_assets.adoc (hunk 1 above)
Comment (aireilly) [reply clusters/bare_assets.adoc]: updated.

**Comment by cadawson** [comment]
Yes, I can take a look at this. Thanks!

**Review by cdoan1 — APPROVED** [review]

**Comment by cdoan1** [comment]
LGTM

**Comment by aireilly** [comment]
@cadawson This should be ready for you now, final peer review + merge :)

File: clusters/bare_assets.adoc [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
+Bare metal assets are virtual or physical servers that you configure to run your {ocp-short} clusters.
+{product-title} connects to a bare metal asset that your administrator creates. Then you can deploy the bare metal asset in a managed cluster.
 
-You must create a bare metal asset in {product-title} to create a cluster on it.
-Use the following procedure to create a bare metal asset that can host a cluster that is managed by {product-title}.
+The hub cluster inventory controller defines a custom resource definition (CRD) called `BareMetalAsset`, that holds the bare metal asset inventory record. When provisioning a managed cluster, the inventory controller reconciles `BareMetalAsset` inventory records with corresponding `BareMetalHost` resources in the managed cluster.
+
+{product-title-short} uses `BareMetalAsset` CR's to provision cluster hardware based on records entered in a configuration management database (CMDB) or similar system. An external tool or automation polls the CMDB and uses the {product-title-short} API to create corresponding `BareMetalAsset` and corresponding `Secret` resources in a hub cluster for subsequent deployment in a managed cluster.
```
Comment (cadawson) [inline clusters/bare_assets.adoc]: Remove apostrophe in CR's.

**Review by aireilly — COMMENTED** [review]
Thanks Chris!

**Review by cadawson — APPROVED** [review]
Comments addressed.

**Pull request merged**